.github/workflows/core-web.deploy.test.yaml

```diff
@@ -0,0 +1,28 @@
+name: CORE WEB - Promote TEST
+
+on:
+  workflow_dispatch:
+
+env:
+  ORIG_TAG: dev
+  PROMOTE_TAG: test
+
+jobs:
+  promote-image-to-test:
+    name: promote-core-web-to-test
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install oc
+        uses: redhat-actions/openshift-tools-installer@v1
+        with:
+          oc: "4.7"
+
+      - name: oc login
+        run: |
+          oc login --token=${{ secrets.BUILD_TOKEN }} --server=${{ secrets.CLUSTER_API }}
+
+      - name: Promote core-web from dev to test
```

Comment:
How does the roll out happen after promotion? 

The gitops projects / trigger are missing. So is the roll out.

Reply:
Right. Adding that now.